### PR TITLE
HL-142

### PR DESCRIPTION
**Pull request by brettvniekerk**
[Link to ticket](https://dltx.atlassian.net/jira/software/projects/HL/boards/82?selectedIssue=HL-142)

- Adds `Abandoned` race status

### CODE REVIEW

**Review by OptimusOpus — APPROVED** [review]

File: src/types/meets.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ export type Race = {
   end_unix?: number;
   close?: string;
   close_unix?: number;
-  status: "Normal" | "Interim" | "Paying";
+  status: "Normal" | "Interim" | "Paying" | "Abandoned";
```
Comment (bitcoinbrisbane) [inline src/types/meets.ts:28]: Lets order this way...

```suggestion
  status: "Abandoned" | "Interim" | "Normal" | "Paying" ;
```

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
One suggestion

File: src/components/Dashboard/DashboardTable.tsx [hunk 2]
```diff
@@ -57,7 +57,8 @@ export const DashboardTable: React.FC<Props> = ({ meets }) => {
                             <div
                               className={`px-3 py-4 whitespace-nowrap text-sm
                                 ${
-                                  race.status === "Paying"
+                                  race.status === "Paying" ||
+                                  race.status === "Abandoned"
                                     ? "bg-gray-400 hover:bg-gray-500"
                                     : race.status === "Interim"
                                     ? "bg-emerald-400"
```
Comment (pondpiu) [inline src/components/Dashboard/DashboardTable.tsx:64]: use classnames?

Comment (brettvniekerk) [reply src/components/Dashboard/DashboardTable.tsx:64]: good idea

File: src/types/meets.ts (hunk 1 above)
Comment (brettvniekerk) [reply src/types/meets.ts:28]: will do

**Review by pondpiu — APPROVED** [review]

**Review by OptimusOpus — APPROVED** [review]

**Review by bitcoinbrisbane — APPROVED** [review]

**Pull request merged**